### PR TITLE
TF 1.1.11 Docs Update

### PR DESCRIPTION
**Pull request by darren-rafay**
Added AKS Cluster v3 update, Tag Group, and Project Tags Association.

### CODE REVIEW

File: docs/resources/projecttagsassociation.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rafay_projecttagsassociation Resource - terraform-provider-rafay"
```
Comment (kutumba-rafay) [inline docs/resources/projecttagsassociation.md:3]: change rafay_projecttagsassociation to rafay_project_tags_association

File: docs/resources/projecttagsassociation.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+subcategory: ""
+description: |-
+  
+---
+
+# rafay_projecttagsassociation (Resource)
```
Comment (kutumba-rafay) [inline docs/resources/projecttagsassociation.md:9]: change rafay_projecttagsassociation to rafay_project_tags_association

File: docs/resources/projecttagsassociation.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+- `timeouts` - (Block) Sets the duration of time the create, delete, and update functions are allowed to run. If the function takes longer than this, it is assumed the function has failed. The default is 10 minutes. (See [below for nested schema](#nestedblock--timeouts))
+
+<a id="nestedblock--metadata"></a>
+### Nested Schema for `metadata`
+
+***Required***
+
+- `name` - (String) The name of the project tag association. This must be unique in your organization.
+- `project` - (String) The name of the Rafay project the cluster will be created in.
```
Comment (kutumba-rafay) [inline docs/resources/projecttagsassociation.md:48]: The name of the Rafay project the cluster will be created in should be changed to The name of the Rafay project the project tag association will be created in

File: docs/resources/projecttagsassociation.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+- `association` - (Map of String) The tags to attach to the project tag association. (See [below for nested schema](#nestedblock--spec--association))
+
+<a id="nestedblock--spec--association"></a>
+### Nested Schema for `spec.assocation`
+
+***Required***
+
+- `tag_key` - (String) The tag key.
+- `tag_type` - (String) The tag type.
```
Comment (kutumba-rafay) [inline docs/resources/projecttagsassociation.md:67]: Need to add valid values for tag type

File: docs/resources/projecttagsassociation.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+### Nested Schema for `metadata`
+
+***Required***
+
+- `name` - (String) The name of the project tag association. This must be unique in your organization.
+- `project` - (String) The name of the Rafay project the cluster will be created in.
+
+***Optional***
+
+- `labels` - (Map of String) The labels for the cluster.
```
Comment (kutumba-rafay) [inline docs/resources/projecttagsassociation.md:52]: It should be labels for project tag association

File: docs/resources/taggroup.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rafay_taggroup Resource - terraform-provider-rafay"
```
Comment (kutumba-rafay) [inline docs/resources/taggroup.md:3]: change rafay_taggroup to rafay_tag_group

File: docs/resources/taggroup.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+subcategory: ""
+description: |-
+  
+---
+
+# rafay_taggroup (Resource)
```
Comment (kutumba-rafay) [inline docs/resources/taggroup.md:9]: change rafay_taggroup to rafay_tag_group

File: docs/resources/taggroup.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+- `timeouts` - (Block) Sets the duration of time the create, delete, and update functions are allowed to run. If the function takes longer than this, it is assumed the function has failed. The default is 10 minutes. (See [below for nested schema](#nestedblock--timeouts))
+
+<a id="nestedblock--metadata"></a>
+### Nested Schema for `metadata`
+
+***Required***
+
+- `name` - (String) The name of the tag group. This must be unique in your organization.
+- `project` - (String) The name of the Rafay project the cluster will be created in.
```
Comment (kutumba-rafay) [inline docs/resources/taggroup.md:48]: replace cluster with tag group

File: docs/resources/taggroup.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+***Optional***
+
+- `labels` - (Map of String) The labels for the cluster.
```
Comment (kutumba-rafay) [inline docs/resources/taggroup.md:52]: replace cluster with tag group

File: docs/resources/taggroup.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+<a id="nestedblock--spec"></a>
+### Nested Schema for `spec`
+
+***Required***
+
+- `tags` - (Map of String) Tags are key/value pairs that are attached to the object.
```
Comment (kutumba-rafay) [inline docs/resources/taggroup.md:59]: @stephan-rafay is tags a "Map of String" or a "Block list"

**Review by stephan-rafay — APPROVED** [review]

**Pull request merged**